script/setup.js

```diff
@@ -167,8 +167,36 @@ try {
 		}
 	};
 
+	const mainContributions = [
+		"code",
+		"content",
+		"doc",
+		"ideas",
+		"infra",
+		"maintenance",
+		"projectManagement",
+		"tool",
+	];
+
+	const addContributor = (user, contributions = []) =>
+		$`all-contributors add ${user} ${contributions.join(",")}`;
```

Comment:
[Refactor] Why separate these out into an earlier place in code if `addContributor` is only ever called once? Why not have the <code>await $\`...\`</code> inline in the one usage place?

```js
await $`all-contributors add ${user} ${[ /* ... */ ].join(",")}`;
```